python/ray/autoscaler/_private/aws/config.py

```diff
@@ -11,7 +11,8 @@
 
 import boto3
 import botocore
-
+from botocore.exceptions import ClientError
```

Comment:
Looks like botocore is already imported in L13?

Reply:
Yup, thanks for catching this